.github/workflows/release.yml

```diff
@@ -197,15 +197,17 @@ jobs:
             version: canonical_version,
           });
 
-          if (isLatest) {
+          console.log("Latest version?", isLatest);
+
+          if (isLatest === "true") {
             return edition === 'ee' ? 'enterprise/latest' : 'latest';
           }
           return "false";
 
     - name: Upload to S3
       run: aws s3 cp ./metabase.jar s3://${{ vars.AWS_S3_DOWNLOADS_BUCKET }}/${{ steps.version_path.outputs.result }}/metabase.jar
     - name: Upload to s3 latest path
-      if: ${{ steps.is_latest.outputs.result != 'false' }}
+      if: ${{ steps.latest_path.outputs.result != 'false' }}
```

Comment:
caught this while testing - we weren't even passing the path for the `if` correctly 🤦